mne/stats/regression.py

```diff
@@ -134,3 +134,156 @@ def _fit_lm(data, design_matrix, names):
         mlog10_p_val[predictor] = -np.log10(p_val[predictor])
 
     return beta, stderr, t_val, p_val, mlog10_p_val
+
+
+def regress_continuous(raw, events, event_id=None,
```

Comment:
I've started liking regress_contionuous because it patterns nicely with Tals and Denis' linear_regression; they're both linear, but linear_regression is for epochs and regress_continuous for, well, continuous data.